modules/@apostrophecms/piece-type/index.js

```diff
@@ -577,13 +577,6 @@ module.exports = {
         });
       },
 
-      // Implementation of the PATCH route. Factored as a method to allow
```

Comment:
I moved this back up to the route because there is no longer a generic PATCH route so it was carrying water for no one.